uefi-test-runner/src/proto/misc.rs

```diff
@@ -0,0 +1,60 @@
+use uefi::prelude::*;
+use uefi::proto::misc::ResetNotification;
+use uefi::table::runtime;
+
+pub fn test(bt: &BootServices) {
+    test_reset_notification(bt);
+}
+
+pub fn test_reset_notification(bt: &BootServices) {
+    info!("Running loaded ResetNotification protocol test");
+
+    let handle = bt.get_handle_for_protocol::<ResetNotification>();
```

Comment:
Call `unwrap` or `expect` here rather than just warning on failure, we want the test-runner to fail if a protocol that should exist is missing.